pages/linux/pacdiff.md

```diff
@@ -0,0 +1,29 @@
+# pacdiff
+
+> Pacorig, pacnew and pacsave maintenance utility for pacman.
+> If you want to use a different program for diffs, set the `DIFFPROG` enviromental variable to point to your desired binary, with any needed flags. `vim -d` is the default.
+> More information: <https://man.archlinux.org/man/pacdiff.8>.
```

Comment:
```suggestion
> More information: <https://manned.org/pacdiff>.
```

Reply:
I'm okay with using the Arch Linux man pages for this one since this is a pretty Arch-centric command. Would be fine with using `manned.org` as well. 😄

Reply:
If manned is prefered that's fine by me -- should I use manned or archlinux?